DependencyInjection/BernardExtension.php

```diff
@@ -130,4 +134,17 @@ private function registerListeners(array $config, ContainerBuilder $container)
             }
         }
     }
+
+    public function configureProfiler(ContainerBuilder $container, array $config)
+    {
+        if (isset($config['profiler']['enabled']) && !$config['profiler']['enabled']) {
+            return;
+        }
+
+        $definition = new Definition(ProducerCollector::class);
```

Comment:
I did indeed loaded the xml in an earlier version, but now I indeed use a php configuration. Or did I miss something?